Resources/Prototypes/_EE/Entities/Objects/Weapons/Melee/terminus.yml

```
@@ -0,0 +1,52 @@
+    soundHit:
+      collection: MetalThud
+  - type: DamageOtherOnHit
+    staminaCost: 16
+  - type: Wieldable
+  - type: GunRequiresWield
+  - type: Item
+    size: Ginormous
+  - type: DisarmMalus
+  - type: Tool
+    qualities:
+    - Prying
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)